src/services/utils/IssueUtil.ts

```diff
@@ -74,3 +77,102 @@ export function checkIsInAssignFlow(
 
   return hasMentor;
 }
+
+function appendNotification(): string {
+  return `\r\n\r\n-------------------------------\r\n\r\n## :warning:Notification:warning:\r\n\r\nxxx<!-- probot:Notification -->\r\n\r\n`;
+}
+
+function appendStatus(): string {
+  return `\r\n\r\n-------------------------------\r\n\r\n## Status\r\n\r\nxxx<!-- probot:Status -->\r\n\r\n`;
+}
+
+function updateNotification(
```

Comment:
Could you please add tests to your functions?

Reply:
I will add function test and rethink my code. Thank for your patient again.

Reply:
OK, I will add some tests.

Reply:
@hi-rustin Sorry to disturb you on the weekend. I get some problem about this test part.
[createOrUpdateStatus function test](https://github.com/tidb-community-bots/ti-challenge-bot/pull/189#pullrequestreview-536029207)